src/containers/costume-tab.jsx

```diff
@@ -41,27 +75,41 @@ class CostumeTab extends React.Component {
             this.setState({selectedCostumeIndex: target.costumes.length - 1});
         }
     }
-
     handleSelectCostume (costumeIndex) {
         this.props.vm.editingTarget.setCostume(costumeIndex);
         this.setState({selectedCostumeIndex: costumeIndex});
     }
-
     handleDeleteCostume (costumeIndex) {
         this.props.vm.deleteCostume(costumeIndex);
     }
-
     handleNewCostume () {
         if (!this.props.vm.editingTarget) return;
         const costumes = this.props.vm.editingTarget.sprite.costumes || [];
         this.setState({selectedCostumeIndex: Math.max(costumes.length - 1, 0)});
     }
-
+    handleNewBlankCostume () {
+        const emptyItem = costumeLibraryContent.find(item => (
+            item.name === 'Empty'
+        ));
+        const costumes = this.props.vm.editingTarget.sprite.costumes || [];
+        const name = this.props.vm.editingTarget.isStage ? `backdrop${costumes.length}` : `costume${costumes.length}`;
```

Comment:
I think this might be off by 1 from what you want (if you have only backdrop1, it tries to add another backdrop called backdrop1) but it's fixed because vm's addCostume automatically runs StringUtil.unusedName to increment the number.

I think it's safe to always name new backdrops 'backdrop1' and new costumes 'costume1' here, since the string util will automatically handle the number.

Also we probably want internationalization for the word 'backdrop'

Reply:
It was actually incrementing well it seemed to me at first, but good to know that was the vm! I will change it to `costume1` and `backdrop1`